docs/java/java-debugging.md

```diff
@@ -1,15 +1,15 @@
 ---
 Order: 3
 Area: java
-TOCTitle: Debugging
+TOCTitle: Running and Debugging
```

Comment:
This will cause the TOC to wrap which doesn't look very nice. Are you sure you need to make this change?